install_deps.sh

```diff
@@ -91,19 +94,19 @@ then
 		. ./scripts/install_openxr.sh
 	fi
 
-	if [ ! -d "${temp_dir}/gtsam" ] && y_or_n "Next: Install gtsam from source"; then
+	if [ ! -d "${opt_dir}/gtsam" ] && y_or_n "Next: Install gtsam from source"; then
 		. ./scripts/install_gtsam.sh
 	fi
 
-	if [ ! -d "${temp_dir}/opengv" ] && y_or_n "Next: Install opengv from source"; then
+	if [ ! -d "${opt_dir}/opengv" ] && y_or_n "Next: Install opengv from source"; then
 		. ./scripts/install_opengv.sh
 	fi
 
-	if [ ! -d "${temp_dir}/DBoW2" ] && y_or_n "Next: Install DBoW2 from source"; then
+	if [ ! -d "${opt_dir}/DBoW2" ] && y_or_n "Next: Install DBoW2 from source"; then
 		. ./scripts/install_dbow2.sh
 	fi
 
-	if [ ! -d "${temp_dir}/Kimera-RPGO" ] && y_or_n "Next: Install Kimera-RPGO from source"; then
+	if [ ! -d "${opt_dir}/Kimera-RPGO" ] && y_or_n "Next: Install Kimera-RPGO from source"; then
```

Comment:
I went through the [Kimera installation documentation](https://github.com/MIT-SPARK/Kimera-VIO/blob/master/docs/kimera_vio_install.md) and noticed that all the dependencies except RPGO do `make install`. So only RPGO needs to go in `opt`. 
One edge case is:
1. Install gtsam
2. Reboot machine
3. Install opengv

In this case step 3 will fail because opengv needs gtsam's Eigen and it would have been wiped out from `tmp`. So maybe we can keep gtsam in `opt` just to be safe, but `opengv` and `DBoW2` can easily go in `tmp`.
On the other hand, `tmp` is just a disk space saving optimization, so maybe it is not entirely necessary.

Reply:
I think we should just keep everything in opt for kimera for simplicity.